tests/scripts/crossgen_comparison.py

```diff
@@ -0,0 +1,372 @@
+#!  /usr/bin/env python
```

Comment:
In general, I'd like to see more comments. Especially about the scenario this is used in, and the arguments that are passed to it. Maybe with an example.

Reply:
Please add a standard header. A good model is: https://github.com/dotnet/coreclr/blob/master/tests/scripts/lst_creator.py#L1